docs/content/setup/linux.md

```diff
@@ -9,6 +9,15 @@ tags: ["setup", "AppImage", "linux"]
 
 ## Supported operating systems
 
+The Infection Monkey AppImage package is supported on the following operating systems (not an
+exhaustive list):
```

Comment:
Let's provide some information about what an AppImage is (a distribution-agnostic package). We can say it should run on most modern Linux distros that have FUSE installed, but here are the ones we've tested:

Reply:
Updated. WDYT?